Rocket.Chat/SettingsViewModel.swift

```diff
@@ -27,6 +27,14 @@ final class SettingsViewModel {
         return localized("myaccount.settings.license")
     }
 
+    internal var canViewAdminPanel: Bool {
+        guard let user = AuthManager.currentUser(), !user.isInvalidated else { return false }
+        return user.hasPermission(.viewPrivilegedSetting) ||
+            user.hasPermission(.viewStatistics) ||
+            user.hasPermission(.viewUserAdministration) ||
+            user.hasPermission(.viewRoomAdministration)
+    }
+
```

Comment:
We can test this if we make it a method or computed variable of User

user.canViewAdminPanel